content/projects/data-science-specific/k-means-clustering/_index.md

```diff
@@ -28,14 +28,22 @@ title: K-Means Clustering Assignment
 
 ## Pre-requisites
 
-Read through the K-Means Tutorials at {{< contentlink path="topics/data-science-specific/clustering" >}} before starting this project.
+Read through the K-Means Tutorials at [content link](http://syllabus.africacode.net/topics/data-science-specific/clustering/) before starting this project.
 
-## Clustering whisky distilleries according to tasting Profiles
+## About the DataSet
 
-Data: [Whisky Tasting Profiles](whisky.csv)
+Goal: Clustering whisky distilleries according to tasting Profiles
 
-Use K-Means clustering to cluster whisky distilleries by their tasting profile. Use the elbow or silhouette method to find the optimal number of clusters.
+Download the dataset: [Whisky Tasting Profiles](whisky.csv)
 
-To see how successful clustering was, report relevant metrics (e.g. silhouette, adjusted rand index, etc.) and create a plot showing the different distilleries, their classes according to the k-Means clustering, and the distance between points. You can use `sklearn.manifold` to get Euclidean distances between points.
+## Instructions
 
-Describe the main differences between the cluster - what are the factors that differ between classes?
\ No newline at end of file
+1. Use K-Means clustering to cluster whisky distilleries by their tasting profile. Use the elbow or silhouette method to find the optimal number of clusters.
+
+2. To see how successful clustering was, report relevant metrics (e.g. silhouette, adjusted rand index, etc.) and create a plot showing the different distilleries, their classes according to the k-Means clustering, and the distance between points. You can use `sklearn.manifold` to get Euclidean distances between points.
+
+3. Describe the main differences between the cluster - what are the factors that differ between classes?
```

Comment:
Are there many clusters or just one?  Are we asking for the difference between cluster**s** or something else?

Reply:
Multiple clusters are possible within the scope of this machine-learning algorithm. Clustering algorithms such as k-means clustering or hierarchical clustering are frequently employed in data analysis. They automate the task of identifying and grouping data points into clusters, usually guided by a similarity metric. The analyst may predefine the number of clusters or let the algorithm determine it, depending on the specific data and problem at hand. Consequently, it is entirely plausible and often an anticipated outcome to discover multiple clusters when examining data or categorizing objects according to specific criteria. The precise quantity and characteristics of these clusters are contingent upon the unique context and the dataset under examination.

Reply:
What I mean is: Should the sentence say:

3. Describe the main differences between the cluster - what are the factors that differ between classes?

Or

3. Describe the main differences between the clusters - what are the factors that differ between classes?


The second one is a pleural. It seems like we want a pleural?  If you believe a singular is more accurate then I would recommend revising the sentence.

Reply:
Does seem like in this context "cluster" should be "clusters" but I can see that error was in the original project